.github/workflows/release-quickstart.yml

```diff
@@ -0,0 +1,133 @@
+name: Release Quickstart Workflow
+on:
+  pull_request_target:
+    types:
+      - closed
+    branches:
+      - main
+      - release-v*
+      - release-next
+
+jobs:
+  release-quickstart:
+    name: Release Quickstart Job
+    # when the ref is release-next the label "quickstartrelease" must be present on the PR for this job to run, and when
+    # the ref is a release branch the label is not required
+    if: |
+      github.event.pull_request.merged == true
+      && (
+        github.ref != 'refs/heads/release-next'
+        || contains(github.event.pull_request.labels.*.name, 'quickstartrelease')
+      )
+    runs-on: ubuntu-latest
+    env:
+      ZITI_QUICKSTART_IMAGE: ${{ vars.ZITI_QUICKSTART_IMAGE || 'docker.io/openziti/quickstart' }}
+      # use github.ref, not github.head_ref, because this workflow should only run on merged PRs in the target/base
+      # branch context, not the PR source branch
+      GITHUB_REF:            ${{ github.ref }}
+      # user github.sha, not github.pull_request.head.sha, because this workflow should only run on merged PRs in the
+      # target/base branch, not the PR source branch
+      GITHUB_SHA:            ${{ github.sha }}
+    steps:
+      - name: Debug action
+        uses: hmarr/debug-action@v2.1.0
+
+      - name: Wait for other builds to complete
```

Comment:
what are "other builds"? i don't understand why this is needed at all? shouldn't this action only trigger after these builds are done???